database/utils/async-for-each.js

```diff
@@ -1,20 +1,20 @@
 require('dotenv').config();
 
-const { WAITTIME_BETWEEN_CALLS } = process.env;
-const waitBetweenCalls =  Number(WAITTIME_BETWEEN_CALLS);
+const WAITTIME_BETWEEN_CALLS = process.env.WAITTIME_BETWEEN_CALLS || 1;
+const waitBetweenCalls = Number(WAITTIME_BETWEEN_CALLS);
 
 const wait = (seconds) => {
   seconds === 0 ? 1 : seconds;
   return new Promise(resolve => {
     setTimeout(() => {
       resolve(seconds);
```

Comment:
```suggestion
      resolve();
```
We don't use the return value, so this can be empty.